crypto/evp/evp_pkey.c

```diff
@@ -18,7 +18,8 @@
 
 /* Extract a private key from a PKCS8 structure */
 
-EVP_PKEY *EVP_PKCS82PKEY(const PKCS8_PRIV_KEY_INFO *p8)
+EVP_PKEY *evp_pkcs82pkey_int(const PKCS8_PRIV_KEY_INFO *p8, OPENSSL_CTX *libctx,
+                             const char *propq)
```

Comment:
So while you're making new public functions out of the others, this one you're making internal...  it seems inconsistent, wanna tell me why?

Reply:
It wasn't clear to me whether we wanted to encourage the use of `EVP_PKCS82PKEY` or not. While I wasn't sure I erred on the side of caution thinking it was easier to make things public later than the other way around.